### PR TITLE
Limit reserved tasks

### PR DESCRIPTION
**Pull request by dmorina**
limits the max number of reserved tasks to 8, added permissions and error messages to the task feed
sets toast position to fixed

### CODE REVIEW

File: crowdsourcing/viewsets/task.py [hunk 1]
```diff
@@ -123,22 +128,24 @@ def list_by_status(self, request, *args, **kwargs):
         for key, value in status_map.iteritems():
             task_workers = TaskWorker.objects.filter(worker=request.user.userprofile.worker, task_status=key)
             serializer = TaskWorkerSerializer(instance=task_workers, many=True,
```
Comment (nistala) [inline crowdsourcing/viewsets/task.py]: getting a permission error here with existing data on dashboard - "You do not have permission to perform this action."

Comment (dmorina) [inline crowdsourcing/viewsets/task.py]: checking ...

**Comment by dmorina** [comment]
@nistala check again, fixed the return value

File: crowdsourcing/permissions/task.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+from crowdsourcing.models import TaskWorker
+from rest_framework.exceptions import PermissionDenied
+
+class HasExceededReservedLimit(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        if view.action == 'create' and TaskWorker.objects.filter(worker=request.user.userprofile.worker,
+                                                                 task_status__in=[1, 5]).count() > 8:
```
Comment (nistala) [inline crowdsourcing/permissions/task.py]: should be >= 8

**Comment by nistala** [comment]
@dmorina one comment otherwise lgtm

**Comment by dmorina** [comment]
@nistala cool, changed, all should be well now

**Comment by neilthemathguy** [comment]
lgtm++

**Pull request merged**